api/secret/create.go

```diff
@@ -259,5 +255,14 @@ func CreateSecret(c *gin.Context) {
 		return
 	}
 
+	l.WithFields(logrus.Fields{
+		"secret_id":   s.GetID(),
+		"secret_name": s.GetName(),
+		"secret_org":  s.GetOrg(),
+		"secret_repo": s.GetRepo(),
+		"secret_type": s.GetType(),
+		"secret_team": s.GetTeam(),
```

Comment:
do we need to check type and `delete` team like the other funcs? doesnt seem necessary but nice to have 🤷

Reply:
fixed in https://github.com/go-vela/server/pull/1145/commits/76997452b4a41417c00847b7b485fc1d0df3bda9